Composer/packages/client/__tests__/testUtils/sampleDialog.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Maybe moving this file inside \_\_mocks folder rather than directly inside testUtils

Reply:
That's a good idea.